cypress.config.js

```diff
@@ -0,0 +1,17 @@
+require("dotenv").config();
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  chromeWebSecurity: false,
+  e2e: {
+    video: false,
+    setupNodeEvents(on, config) {
+      config.env = {
+        baseUrl: "http://127.0.0.1:5500/",
```

Comment:
You can actually omit this, since Cypress runs its own webserver at the root just like live-server. So by default it will try to run whatever package is at the root.